apps/chat-android/build.gradle

```diff
@@ -16,7 +16,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.android.tools.build:gradle:4.0.0'
```

Comment:
why downgraded?

Reply:
AS 4.0 does not support gradle 4.2.2, but Android Studio 4.0 and later is stated as requirement in the blog post.